src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -24,15 +24,15 @@ export default class TwitterHandleLanding extends Component {
   constructor (props) {
     super(props);
     this.state = {
-      twitter_handle: "",
+      twitterHandle: "",
     };
     this.getIncomingActiveRoute = this.getIncomingActiveRoute.bind(this);
   }
 
   componentDidMount () {
-    // console.log("TwitterHandleLanding componentDidMount, this.props.params.twitter_handle: " + this.props.params.twitter_handle);
-    this.setState({ twitter_handle: this.props.params.twitter_handle });
-    TwitterActions.twitterIdentityRetrieve(this.props.params.twitter_handle);
+    // console.log("TwitterHandleLanding componentDidMount, this.props.params.twitterHandle: " + this.props.params.twitterHandle);
+    this.setState({ twitterHandle: this.props.params.twitterHandle });
```

Comment:
Hi Fu-Lin, we cannot change the case of incoming this.props.params variables, so this should be reverted to `this.props.params.twitter_handle`. It can be passed into a state variable `this.state.twitterHandle` though.